cmd/legacy_main.go

```diff
@@ -76,7 +77,12 @@ func registerTerminatingSignalHandler(mountPoint string, c *cfg.Config) {
 
 			err := fuse.Unmount(mountPoint)
 			if err != nil {
-				logger.Errorf("Failed to unmount in response to %s: %v", sigName, err)
+				if errors.Is(err, fuse.ErrExternallyManagedMountPoint) {
+					logger.Infof("Mount point %s is externally managed; gcsfuse will not unmount it.", mountPoint)
+					return
+				} else {
+					logger.Errorf("Failed to unmount in response to %s: %v", sigName, err)
+				}
```

Comment:
You can remove the "else" that's below as well.
```suggestion
				}
				logger.Errorf("Failed to unmount in response to %s: %v", sigName, err)
				return
```